sql/core/src/main/scala/org/apache/spark/sql/execution/ui/ExecutionPage.scala

```diff
@@ -128,6 +136,10 @@ class ExecutionPage(parent: SQLTab) extends WebUIPage("execution") with Logging
   private def jobURL(request: HttpServletRequest, jobId: Long): String =
     "%s/jobs/job/?id=%s".format(UIUtils.prependBaseUri(request, parent.basePath), jobId)
 
+  private def stageURL(request: HttpServletRequest, stageId: Int, attemptId: Int): String =
+    "%s/stages/stage/?id=%s&attempt=%s".format(
```

Comment:
nit: since you use `String.format()` you can use `%d` for the Int parameters, but it doesn't really matter

Reply:
Made the change.